### DISCUSSION TITLE
Allow scrubbing through replays by clicking on the pie chart

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#19487] [Ideas]
For me personally, I prefer the pie chart over the previous difficulty graph progress bar but I miss having the functionality of being able to scrub through replays using the progress bar. I think that it would be a cool addition to be able to scrub through replays by clicking and dragging on the pie chart.

**Comment by peppy** [comment]
You can use keyboard controls to scrub as an alternative.

**Comment by frenzibyte** [comment]
One idea I had in mind is to separate the song progress handle away from the default song progress implementation and have it display regardless of skin (and the `HUDOverlay` would deal with positioning it over the default song progress if it still exists).

**Comment by khang06** [comment]
Maybe popping open a hidden seek bar via a keyboard shortcut like McOsu could work?
